nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/ui/common/property-table/editors/nf-editor/nf-editor.component.ts

```diff
@@ -172,6 +172,7 @@ export class NfEditor implements OnDestroy {
             mode: this.mode,
             readOnly: this.readonly,
             lineNumbers: true,
+            theme: 'nifi',
```

Comment:
This adds the `.cm-s-nifi` to any codemirror instance so it will inherit the theme.